samcli/cli/command.py

```diff
@@ -104,6 +96,117 @@ def _set_commands(package_names):
 
         return commands
 
+    def format_options(self, ctx, formatter):
+        # Re-order options so that they come after the commands.
+        self.format_commands(ctx, formatter)
+        opts = [RowDefinition(text="\n")]
+        for param in self.get_params(ctx):
+            rv = param.get_help_record(ctx)
```

Comment:
nit: Can we be a bit more descriptive with the name? From first glance it wasn't 100% intuitive to me what `rv` meant.

Reply:
renaming to `row`